docs/1-introduction/1-1-getting-started/1-1-2-hello-d3d11.md

```diff
@@ -1,8 +1,19 @@
 # Hello D3D11
 
-We setup a base application in the former chapter. We wont be showing `Main.cpp` and `Application.cpp` anymore.
-
-Let's break down the relevant bits and pieces by showing you how the new class, which derives from `Application` will look like.
+In this chapter, we'll introduce you to the basics of using D3D11; how to create a ID3D11Device and
+how to use it to show something in our window. In the last chapter we setup a basic implementation
+for a application with a window through GLFW. The implementation for `Main.cpp` and
```

Comment:
`for an application`

Reply:
I always forget the `n` and I really don't understand why I just got used to always writing `a`.......